boards/samr21-xpro/include/pwm_params.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   boards_samr21-xpro
+ * @{
+ *
+ * @file
+ * @brief     Board specific configuration of direct mapped PWM
+ *
+ * @author    Fracisco Acosta <francisco.acosta@inria.fr>
+ */
+
+#ifndef PWM_PARAMS_H
+#define PWM_PARAMS_H
+
+#include "board.h"
+#include "periph/pwm.h"
+#include "saul/periph.h"
+
+#define MODE        PWM_LEFT
+#define FREQU       (1000U)
+#define STEPS       (1000U)
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief    PWM pin configuration
+ */
+static const  saul_pwm_params_t saul_pwm_params[] =
```

Comment:
I don't think mapping all those PWM lines per default is a good idea, as the pin usage overlaps with other functions as e.g. I2C (PA 16) and the ADC (PA6 and PA7). Also there is IMHO no need to export all 5 PWM pins per default in the first place. I would suggest to only export pins PA18 and PA19, as these per default do not overlap with anything.